File: conveyor/src/main/java/ru/artemev/conveyor/model/enums/Gender.java
package ru.artemev.conveyor.model.enums;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;

@Schema(description = "Пол")
@AllArgsConstructor
public enum Gender {
  MALE("Мужчина"),
  FEMALE("Женщина"),
  NOT_BINARY("Другое");
  private final String description;
}

File: gateway/src/main/kotlin/ru/artemev/gateway/model/enums/Gender.kt
package ru.artemev.gateway.model.enums

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Пол")
enum class Gender(val description: String) {
    MALE("Мужчина"),
    FEMALE("Женщина"),
    NOT_BINARY("Другое")
}